runtime/rococo/src/lib.rs

```diff
@@ -888,6 +888,7 @@ impl pallet_identity_management_mock::Config for Runtime {
 impl pallet_vc_management::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type TEECallOrigin = EnsureEnclaveSigner<Runtime>;
+	type SetAdminOrigin = EnsureSignedBy<ALICE, AccountId>;
```

Comment:
I suggest using `EnsureRootOrHalfCouncil` too, as rococo is a public network. So everyone has access to `ALICE`

Reply:
fixed